src/generateRoutes.js

```diff
@@ -71,6 +79,21 @@ function generateRoutes(routes, app, endpointInfo = false) {
         await currentController(usecase, request, req.user, res, next)
       })
     }
+
+    if (route.other) route.other.forEach((other) => {
+      if (other.REST?.verb && other.REST?.path) {
```

Comment:
Does this refer to how this is exported in metadata?
https://github.com/herbsjs/herbs2rest/blob/aa77d220cc18c34389a9b592402723d384c49658/test/generateControllers.test.js#L99

As for the if, I programmed it so that only one endpoint is generated for the usecase, if it is exported with path and verb metadata, I'm still working on the so resourceName option.

If a usecase has no path or verb, and is of type other, it is likely that the user would not want an endpoint created.

Soon I will work on the standard POST verb for usecases of type other.

Reply:
Perhaps we can wrap the herbarium in this, and pass the verb as` herbarium.verb.{getAll | getById | post | put | delete}` in the usecase export, or `herbarium.other`, but this when exporting the usecase, since in this line we only read the metadata of the found usecases.

Reply:
My point is just that the current UX is `{action} = {path}`, where `{action}` being `getAll | getById | post | put | delete`. The `{action}` will be later translated to a HTTP Verb (`GET`, `POST`, etc)

Reply:
Ok, so instead of `verb: "GET"` we can pass something like:
`herbarium.actions.{getAll | getById | post | put | delete}` ?

Reply:
As mentioned in https://github.com/herbsjs/herbs2rest/pull/49#discussion_r1090069003, it is just 
`other.REST.{getAll | getById | post | put | delete} = '/user/:id/bla/foo'`, no `.actions.` in it.

Reply:
Solved